### ISSUE TITLE
Safari cannot open the page because the address is invalid

### ISSUE THREAD

**Issue by errolcash** [auth0/auth0-oidc-client-net#195]
<img width="382" alt="Screenshot 2021-05-02 at 10 02 58 PM" src="https://user-images.githubusercontent.com/56737933/116836276-e70f1680-ab93-11eb-9aae-748441d51f33.png">

### Describe the problem

I recently updated to BigSur 11.3 and 14.5 for ios and ended up facing the bug already reported #189. I tried using the work around from that issue and i also downgraded the ios simulator and still face the same issue in safari after entering credentials.

### What was the expected behavior?

After login it should of called back to the application

### Reproduction

1. Launch on ios 14.4 after entering credentials for login. attached image error is displayed

**Comment by errolcash** [comment]
adding the below into my info.plist solved it for ios 14.4

```
<key>CFBundleURLTypes</key>
<array>
    <dict>
        <key>CFBundleTypeRole</key>
        <string>None</string>
        <key>CFBundleURLName</key>
        <string>auth0</string>
        <key>CFBundleURLSchemes</key>
        <array>
            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
        </array>
    </dict>
</array>
```

**Comment by errolcash** [comment]
<key>CFBundleURLTypes</key>
<array>
    <dict>
        <key>CFBundleTypeRole</key>
        <string>None</string>
        <key>CFBundleURLName</key>
        <string>auth0</string>
        <key>CFBundleURLSchemes</key>
        <array>
            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
        </array>
    </dict>
</array>

**Comment by errolcash** [comment]
adding the below to my info.plist solved the issue on 14.4

```
<key>CFBundleURLTypes</key>
<array>
    <dict>
        <key>CFBundleTypeRole</key>
        <string>None</string>
        <key>CFBundleURLName</key>
        <string>auth0</string>
        <key>CFBundleURLSchemes</key>
        <array>
            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
        </array>
    </dict>
</array>
```
